tests/app/Services/DependencyServiceTest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Services;
+
+use App\BaseFunctionalTestCase;
+use App\Models\Dependency\Dependency;
+use App\Models\User\User;
+use Carbon\Carbon;
+use LogicException;
+
```

Comment:
Could be worth specifying the dependency names as variables

Reply:
Done!